compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -1244,3 +1245,19 @@ TR_Debug::printa64(TR::FILE *pOutFile, TR::Snippet * snippet)
          TR_ASSERT( 0, "unexpected snippet kind");
       }
    }
+
+bool
+TR_Debug::isBranchToTrampoline(TR::SymbolReference *symRef, uint8_t *cursor, int32_t &distance)
+   {
+   uintptrj_t target = (uintptrj_t)symRef->getMethodAddress();
+   bool requiresTrampoline = false;
+
+   if (_cg->directCallRequiresTrampoline(target, (intptrj_t)cursor))
+      {
+      target = TR::CodeCacheManager::instance()->findHelperTrampoline(symRef->getReferenceNumber(), (void *)cursor);
```

Comment:
This code makes the assumption that if a trampoline is required then it must be to a helper function.  It is not clear from the function name that this is the case.  At the very least you should add a comment to this API that says it should only be used for inquiring about branches to helpers.  Longer term the API itself should change, but that would likely happen as part of broader RAS refactoring.

Reply:
I added the comment to this function.